pom.xml

```diff
@@ -64,14 +64,20 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-gpg-plugin</artifactId>
-                <version>1.5</version>
+                <version>1.6</version>
                 <executions>
                     <execution>
                         <id>sign-artifacts</id>
                         <phase>verify</phase>
                         <goals>
                             <goal>sign</goal>
                         </goals>
+                        <configuration>
+                            <gpgArguments>
+                                <arg>--pinentry-mode</arg>
```

Comment:
https://github.com/actions/setup-java/blob/b53500dabc37db6cc1dcd35059bdcd85df25d140/docs/advanced-usage.md#extra-setup-for-pomxml